src/v/kafka/server/connection_context.cc

```diff
@@ -46,6 +48,17 @@ using namespace std::chrono_literals;
 
 namespace kafka {
 
+namespace {
+static constexpr std::string_view
+  multi_proxy_initial_client_id("__redpanda_mpx");
```

Comment:
should this be a configuration option?

Reply:
i think this can be hardcoded as it shouldn't change